docs/docs/guides/js_apps/test.md

```diff
@@ -132,13 +132,11 @@ The [`CheatCodes`](../../reference/sandbox_reference/cheat_codes.md) class, whic
 
 ### Set next block timestamp
 
-The `warp` method sets the time for next execution, both on L1 and L2. We can test this using an `isTimeEqual` function in a `Test` contract defined like the following:
+Since the rollup time is dependent on what "slot" the block is included in, the way to progress time is to just progress the time of the underlying chain by a number of slots. The duration of a slot can be looked up using `SLOT_DURATION()` in the Rollup (code in Leonidas.sol).
```

Comment:
```suggestion
Since the rollup time is dependent on what "slot" the block is included in, time can be progressed by progressing slots. The duration of a slot is available by calling `SLOT_DURATION()` on the Rollup (code in Leonidas.sol).
```